src/common/form/fields/__tests__/DateField.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import toJSON from 'enzyme-to-json';
+
+import { shallowWithIntl } from '../../../../../app/utils/testUtils';
+import DateField from '../DateField';
+
+describe('DateField', () => {
+  test('renders correctly', () => {
+    const props = {
```

Comment:
You might want to use `globalDateMock` from testUtils here so snapshot wont change daily